src/handlers/sqs-indexer/handle-bridge-event.ts

```diff
@@ -0,0 +1,38 @@
+import {TransactionResponse} from '@ethersproject/abstract-provider'
+
+import SqsService from '../../services/sqs-service'
+import {networkToChainId} from '../../utils/utils'
+import {NetworkMonitor} from '../../utils/network-monitor'
+import {JobIdentifier, PayloadType, SqsMessageBody} from '../../types/sqs'
+
+async function handleBridgeEvents(
+  networkMonitor: NetworkMonitor,
+  transaction: TransactionResponse,
+  network: string,
+  tags: (string | number)[],
+): Promise<void> {
+  const messageBody: SqsMessageBody = {
+    type: PayloadType.HolographProtocol,
```

Comment:
sorta confused. This should have an eventName right? Like `eventName: 'handleBridgeEvent'.

Ah ok i see you moved `eventName` to `jobIdentifier`. 

In hindsight, i think this may be a better organization:


1. eventName: `MintNft || BridgeNFTPreProcess || BridgeCollection` - Think of this as the name of the event in human readable terms. We should be able to look at the json object and able able to understand the flow this event is suppose to be on. I like to think of this as the main filter of the events.
1. eventSignature: `CrossChainMessageSent(bytes32 messageHash)` - This is the actual function signature of the event.
1. jobIdentifier - I would remove this. I see where your head was at, but i don't think we need something like this. I feel like mixing event and job here is a little confusing. Also my gut reaction to seeing ID is to think of an instance. So maybe in the future we may have an event name with multiple job Id that get tracked. In short, i think we can use `eventName` to be the filter for a while before we need anything else.